bedrock/firefox/views.py

```diff
@@ -678,14 +670,7 @@ class FirefoxProductDesktopView(BlogPostsView):
     blog_tags = ['browser', 'featured']
 
     def get_template_names(self):
```

Comment:
Does this method still need to be used here? Could you just do `template_name = 'firefox/products/desktop.html'`?